.github/workflows/ci.yml

```
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    # branches:
+    # - master
+  pull_request:
+  # Run daily at 0:01 UTC
+  schedule:
+  - cron:  '1 0 * * *'
+  workflow_dispatch:
+
+
+jobs:
+  test:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        # Test all on ubuntu, test ends on macos and windows
+        exclude:
+          - os: macos-latest
+            python-version: 3.7
+          - os: macos-latest
+            python-version: 3.8
+          - os: windows-latest
+            python-version: 3.7
+          - os: windows-latest
+            python-version: 3.8
```

I'm not 100% sure this is valid syntax but seems to be worth a try.
```suggestion
          - os: macos-latest
            python-version: [3.7, 3.8]
          - os: windows-latest
            python-version: [3.7, 3.8]
```

---

Sadly, it is not. :upside_down_face: I wish it was!

---

To be clear: This would still run, but GHA won't properly parse the `[3.7. 3.8]` syntax and then it just runs everything making the excludes pointless.

---

Alright thanks for your reply !